legacyman_parser/parse_non_standard_countries.py

```diff
@@ -19,8 +18,10 @@ def extract_non_standard_countries_in_region(soup: BeautifulSoup = None,
     for area_element in soup.find_all('area'):
         href = area_element.get('href')
         country = area_element.get('alt')
+        coords = area_element.get('coords')
+        shape = area_element.get('shape')
         if href.startswith('..'):
             parsed_region_url = urljoin(parsed_url, href)
             NON_STANDARD_COUNTRY_COLLECTION.append(CountryMap(seq.next_value(), country,
```

Comment:
@brahnavank - while the next block of code refers to `RegionMap` I don´t think it´s actually a  region object.  I think that in this instance a plain old URL is being stored in region-map.

But, now that `RegionMap` includes type and cooridinates we should probably change the code below to use a  new data object (that looks like the original RegionMap structure).

I can have a go at this tomorrow.

Reply:
No, I was wrong. We are parsing the map twice - once for standard countries, and once for non-standard countries.